packages/components/src/components/data-table/_data-table-action.scss

```diff
@@ -17,11 +17,12 @@
   //TOOLBAR
   //-------------------------------------------------
   .#{$prefix}--table-toolbar {
-    display: flex;
-    width: 100%;
     background: $ui-01;
+    display: flex;
```

Comment:
just wondering any reason for moving the `display` and `width` rules?

Reply:
Sorry about that, it's a nitpick of mine. I find it easier to find properties if they're alphabetized. I can revert that change if it holds up the PR.

Reply:
that's fine since I don't think we have any guidelines on CSS property ordering